packages/test-ts/src/validators.ts

```diff
@@ -30,3 +66,34 @@ export const MockFunctionResponseValidator = z.union([z.string(), ParameterizedR
 export const MockSectionValidator = z.record(MockFunctionResponseValidator)
 
 export const MockConfigValidator = z.record(z.union([MockSectionValidator, InputsValidator]))
+
+export const GetPriceRequestValidator = OracleTokenPriceRequestValidator.omit({ timestamp: true }).extend({
```

Comment:
Can we change it to the name of the oracle validator without the Oracle, for instance

GetPriceRequestValidator -> TokenPriceRequestValidator